### PR TITLE
3D ResNet 번역

### PR DESCRIPTION
**Pull request by cpprhtn**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #62 

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._

3D ResNet 번역

### CODE REVIEW

File: facebookresearch_pytorchvideo_resnet.md [hunk 1]
```diff
@@ -119,52 +119,51 @@ try: urllib.URLopener().retrieve(url_link, video_path)
 except: urllib.request.urlretrieve(url_link, video_path)
 ```
 
-Load the video and transform it to the input format required by the model.
+비디오를 불러오고 모델에 필요한 입력 형식으로 변환합니다.
```
Comment (Lee-JaeWon) [inline facebookresearch_pytorchvideo_resnet.md:122]: `영상을 불러오고 이것을 모델에 필요한 입력 형식으로 변환합니다.`
라고 정확히 영상을 변환한다는 의미를 강조해주는 것은 어떤지 의견남겨봅니다!

File: facebookresearch_pytorchvideo_resnet.md [hunk 2]
```diff
@@ -119,52 +119,51 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
+# 예측된 클래스를 레이블 이름에 매핑합니다.
 pred_class_names = [kinetics_id_to_classname[int(i)] for i in pred_classes]
 print("Top 5 predicted labels: %s" % ", ".join(pred_class_names))
 ```
 
-### Model Description
-The model architecture is based on [1] with pretrained weights using the 8x8 setting
-on the Kinetics dataset. 
+### 모델 설명
+모델 아키텍처는 Kinetics 데이터셋의 8x8 설정을 사용하여 사전 훈련된 가중치가 있는 [1]을 기반으로 합니다.
```
Comment (Lee-JaeWon) [inline facebookresearch_pytorchvideo_resnet.md:161]: `모델 아키텍처는 Kinetics 데이터셋의 8x8 설정을 사용하여 사전 훈련된 가중치가 있는 참고문헌 [1]을 기반으로 합니다.`
와 같이 `[1]`에 대해서 참고문헌인 것을 언급해주면 어떤지 의견남겨봅니다!!

**Review by Lee-JaeWon — COMMENTED** [review]
전체적으로 번역이 매끄러운 것 같습니다!! 👍👍

File: facebookresearch_pytorchvideo_resnet.md (hunk 1 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md:122]: `이것`이라는 단어가 들어가니까 의미가 확실해지네요!

File: facebookresearch_pytorchvideo_resnet.md (hunk 2 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md]: 좋은의견 감사합니다 :)

File: facebookresearch_pytorchvideo_resnet.md [hunk 3]
```diff
@@ -119,52 +119,51 @@ try: urllib.URLopener().retrieve(url_link, video_path)
 except: urllib.request.urlretrieve(url_link, video_path)
 ```
 
-Load the video and transform it to the input format required by the model.
+비디오를 불러오고 모델에 필요한 입력 형식으로 변환합니다.
 
 ```python
-# Select the duration of the clip to load by specifying the start and end duration
-# The start_sec should correspond to where the action occurs in the video
+# 시작 및 종료 기간을 지정하여 불러올 클립의 기간을 선택합니다.
```
[the remark this answers is not in the document]
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md:125]: 꼼꼼한 지적 감사합니다 👍

File: facebookresearch_pytorchvideo_resnet.md [hunk 4]
```diff
@@ -119,52 +119,51 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
 inputs = video_data["video"]
 inputs = inputs.to(device)
 ```
 
-#### Get Predictions
+#### 예측값 구하기
 
 ```python
-# Pass the input clip through the model
+# 모델을 통해 입력클립을 전달합니다.
```
Comment (Taeyoung96) [inline facebookresearch_pytorchvideo_resnet.md:147]: #57 을 리뷰할 때도 언급 드렸지만 `입력 클릭`으로 바꾸는건 어떠실까요?

**Review by Taeyoung96 — COMMENTED** [review]
@cpprhtn   
번역 하시느라 고생 많으셨습니다.

File: facebookresearch_pytorchvideo_resnet.md [hunk 5]
```diff
@@ -47,18 +47,18 @@ from pytorchvideo.transforms import (
...
 ```python 
-# Set to GPU or CPU
+# GPU 또는 CPU 방식을 설정합니다.
 device = "cpu"
 model = model.eval()
 model = model.to(device)
 ```
 
-Download the id to label mapping for the Kinetics 400 dataset on which the torch hub models were trained. This will be used to get the category label names from the predicted class ids.
+토치 허브 모델이 훈련된 Kinetics 400 데이터셋에 대해 ID에서의 레이블 매핑 정보를 다운로드합니다. 이는 예측된 클래스 ID에서 카테고리 레이블 이름을 가져오는데 사용됩니다.
```
Comment (hyoyoung) [inline facebookresearch_pytorchvideo_resnet.md:61]: 레이블 매핑 정보 보다는
레이블과 맞는 정보 정도가 어떨까요?

File: facebookresearch_pytorchvideo_resnet.md [hunk 6]
```diff
@@ -71,13 +71,13 @@ except: urllib.request.urlretrieve(json_url, json_filename)
 with open(json_filename, "r") as f:
     kinetics_classnames = json.load(f)
 
-# Create an id to label name mapping
+# 레이블 이름 매핑에 대한 ID 만들기
```
Comment (hyoyoung) [inline facebookresearch_pytorchvideo_resnet.md:74]: 위와 비슷하게
레이블 이름 매핑 보다는
레이블 이름과 맞는 ID 만들기 정도는 어떨까요?

File: facebookresearch_pytorchvideo_resnet.md [hunk 7]
```diff
@@ -119,52 +119,51 @@ try: urllib.URLopener().retrieve(url_link, video_path)
...
 ```
 
-### Model Description
-The model architecture is based on [1] with pretrained weights using the 8x8 setting
-on the Kinetics dataset. 
+### 모델 설명
+모델 아키텍처는 Kinetics 데이터셋의 8x8 설정을 사용하여 사전 훈련된 가중치가 있는 참고문헌 [1]을 기반으로 합니다.
 | arch | depth | frame length x sample rate | top 1 | top 5 | Flops (G) | Params (M) |
-| --------------- | ----------- | ----------- | ----------- | ----------- | ----------- |  ----------- | ----------- |
+| --------------- | ----------- | ----------- | ----------- | ----------- | ----------- |  ----------- |
```
Comment (hyoyoung) [inline facebookresearch_pytorchvideo_resnet.md:163]: markdown 한 컬럼이 지워졌습니다 ----------- |
혹시 이유가 있어서 지우신걸까요?

**Review by hyoyoung — CHANGES_REQUESTED** [review]
전반적으로 잘되어있습니다
몇가지 사항 확인 부탁드립니다

File: facebookresearch_pytorchvideo_resnet.md (hunk 7 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md:163]: hub.org 쪽에서부터 존재했던 오타였는데 저부분을 지워야 column수가 일치합니다!!

File: facebookresearch_pytorchvideo_resnet.md (hunk 6 above)
Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md:74]: 훨씬 자연스러운 것 같습니다. 피드백 감사합니다

File: facebookresearch_pytorchvideo_resnet.md (hunk 7 above)
Comment (hyoyoung) [reply facebookresearch_pytorchvideo_resnet.md:163]: 그럼 혹시 hub.org에 PR은 보내보셨나요?

Comment (cpprhtn) [reply facebookresearch_pytorchvideo_resnet.md:163]: 네네 머지되었습니다 :)

Comment (hyoyoung) [reply facebookresearch_pytorchvideo_resnet.md:163]: good

**Review by hyoyoung — APPROVED** [review]
good
